css/main.css

```diff
@@ -1,3 +1,733 @@
-/* Add your CSS here */
+/* CCS Variables */
+:root {
+  --red_brown: #a05841;
+  --green_black: #133032;
+  --light_grey: #ededee;
+  --white: #ffffff;
+  --off_white: #fafafb;
+  --transparent_white: rgba(0, 0, 0, 0.75);
+  --heading_font-family: "Fjalla One", "Roboto", sans-serif;
+  --body_subheading_font-family: "Outfit", "Roboto", sans-serif;
+}
+
+html {
+  box-sizing: border-box;
+  color: var(--red_brown);
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: var(--light_grey);
+  font-family: var(--body_subheading_font-family);
```

Comment:
The browser automatically adds an 8px margin all around the body. Remove this by setting margin to 0 here - this will make your elements 'bleed off the page', meaning they will span the full width from left to right, like in the design.